Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/ImageSubresource.h.orig

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Why is a copy of the original `ImageSubresource.h` being added?

Reply:
My mistake, should not have committed these files which came from applying a patch. I removed them.